diff --git a/gfx/drivers_context/drm_go2_ctx.c b/gfx/drivers_context/drm_go2_ctx.c
index 856aa0f9a2..1e3d0da50d 100644
--- a/gfx/drivers_context/drm_go2_ctx.c
+++ b/gfx/drivers_context/drm_go2_ctx.c
@@ -346,7 +346,7 @@ static void gfx_ctx_go2_drm_swap_buffers(void *data)
          surf,
          src_x, src_y, src_w, src_h,
          out_y, out_x, out_h, out_w,
-         GO2_ROTATION_DEGREES_270, 2);
+         GO2_ROTATION_DEGREES_90, 2);
    go2_context_surface_unlock(drm->context, surf);
 #endif
 }